.github/workflows/ci.yml

```
@@ -0,0 +1,25 @@
+name: ci
+
+on:
+  push:
+    branches-ignore:
+      - "master"
+      - "develop"
+      - "release/**"
+  pull_request:
+
+jobs:
+  test:
+    name: test
+    strategy:
+      matrix:
+        os: [ ubuntu-20.04, macOs-latest, windows-latest ]
```

```suggestion
        os: [ ubuntu-latest, macOs-latest, windows-latest ]
```
There is not need to use 20.04. In addition, 20.04 will become latest soon.